includes/pages/user_shifts.php

```diff
@@ -374,20 +368,23 @@ function get_ids_from_array($array)
  * @param array  $selected
  * @param string $name
  * @param string $title
- * @param array  $additionalButtons
+ * @param int[]  $ownAngelTypes
  * @return string
  */
-function make_select($items, $selected, $name, $title = null, $additionalButtons = [])
+function make_select($items, $selected, $name, $title = null, $ownAngelTypes = [])
```

Comment:
As this is a generic function imho the wording should be kept generic so in that case eventually `$ownSelect`

Reply:
fixed